crypto-ffi/src/generic/context/e2ei.rs

```diff
@@ -138,15 +138,9 @@ impl CoreCryptoContext {
             .into())
     }
 
-    /// See [core_crypto::context::CentralContext::retain_only_key_packages_of_most_recent_x509_credentials]
-    pub async fn retain_only_key_packages_of_most_recent_x509_credentials(
-        &self,
-        ciphersuite: Ciphersuite,
-    ) -> CoreCryptoResult<()> {
-        Ok(self
-            .context
-            .retain_only_key_packages_of_most_recent_x509_credentials(ciphersuite.into())
-            .await?)
+    /// See [core_crypto::context::CentralContext::delete_stale_key_packages]
+    pub async fn delete_stale_key_packages(&self, ciphersuite: Ciphersuite) -> CoreCryptoResult<()> {
+        Ok(self.context.delete_stale_key_packages(ciphersuite.into()).await?)
```

Comment:
Nit: it's not critical here (or ever), but I don't like the pattern where a function's last statement is 

```rust
Ok(
  some
    .big()
    .function()
    .chain()?
)
```

Just inlining the error handling makes it possible to read the entire thing forward, instead of needing to jump back at the end to get to the `Ok`:

```rust
some.big().function().chain().map_err(Into::into)
```

I've been rewriting these instances as I encounter them, and I'd encourage everyone else to also.